src/components/ProfilePage/ProfileAvatar.jsx

```diff
@@ -32,58 +39,90 @@ class ProfileAvatar extends React.Component {
     this.props.onSave(new FormData(this.form.current));
   }
 
-  render() {
-    const {
-      src,
-    } = this.props;
+  toggleDropdown() {
+    this.setState({
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
 
+  renderPending() {
     return (
-      <div className="profile-avatar-wrap position-relative">
-        <div className="profile-avatar rounded-circle overflow-hidden bg-dark">
-          {this.props.savePhotoState === 'pending' ? (
-            <div
-              className="p-absolute w-100 h-100 d-flex justify-content-center align-items-center"
-              style={{ backgroundColor: 'rgba(255,255,255,.5)' }}
-            >
-              <Spinner color="primary" />
-            </div>
-          ) : null}
-
-          <button
-            className="text-white profile-avatar-edit-button"
-            onClick={this.onClick}
-          >
-            Change
-          </button>
+      <div
+        className="position-absolute w-100 h-100 d-flex justify-content-center align-items-center rounded-circle"
+        style={{ backgroundColor: 'rgba(0,0,0,.65)' }}
+      >
+        <Spinner color="primary" />
+      </div>
+    );
+  }
 
-          <form
-            ref={this.form}
-            onSubmit={this.onSubmit}
-            encType="multipart/form-data"
-          >
-            <img className="w-100" src={src} alt="profile avatar" />
-
-            {/* The name of this input must be 'file' */}
-            <Input
-              className="d-none"
-              innerRef={this.fileInput}
-              type="file"
-              name="file"
-              id="exampleFile"
-              onInput={this.onInput}
-              accept=".jpg, .jpeg, .png"
+  renderMenu() {
+    return (
+      <div className="profile-avatar-menu-container">
+        {this.props.src == null ? (
```

Comment:
Nit?: `===` ?

Reply:
I think this should match undefined as well